ACE/ace/config-integritySCA.h

```diff
@@ -84,65 +116,54 @@
 #define ACE_LACKS_MKSTEMP
 #define ACE_LACKS_MPROTECT
 #define ACE_LACKS_MUTEXATTR_PSHARED
+#define ACE_LACKS_PIPE
 #define ACE_LACKS_RLIMIT
-#define ACE_LACKS_RECVMSG
 #define ACE_LACKS_RWLOCK_T
 #define ACE_LACKS_SEMBUF_T
 #define ACE_LACKS_UNIX_DOMAIN_SOCKETS
-#define ACE_LACKS_FCNTL
+#define ACE_LACKS_USER
+#define ACE_LACKS_FILE_FCNTL
+#define ACE_LACKS_SHARED_MEMORY
 #define ACE_LACKS_UMASK
 #define ACE_LACKS_SEEK
-#define ACE_LACKS_MSYNC
-#define ACE_LACKS_PID_STUFF
+#define ACE_LACKS_IPC_H
+#define ACE_LACKS_SEEK
+#define ACE_LACKS_SHARED_MEMORY
 #define ACE_LACKS_NETDB_REENTRANT_FUNCTIONS
 #define ACE_LACKS_SETGID
 #define ACE_LACKS_PIPE
 #define ACE_LACKS_SYS_PARAM_H
 #define ACE_LACKS_SYS_MSG_H
 #define ACE_LACKS_UTSNAME_T
-#define ACE_LACKS_UNAME
 #define ACE_LACKS_UMASK
 #define ACE_LACKS_ISATTY
 #define ACE_LACKS_GETOPT
 #define ACE_LACKS_STRCASECMP
 #define ACE_LACKS_TRUNCATE
-#define ACE_LACKS_PWD_FUNCTIONS
-#define ACE_LACKS_UNIX_SIGNALS
-#define ACE_HAS_THREAD_SPECIFIC_STORAGE
-#define ACE_LACKS_SYSV_SHMEM
 #define ACE_LACKS_PUTENV
 #define ACE_HAS_4_4BSD_SENDMSG_RECVMSG
 #define ACE_HAS_CLOCK_GETTIME
 #define ACE_HAS_CLOCK_SETTIME
-#define ACE_LACKS_THREAD_PROCESS_SCOPING
-#define ACE_LACKS_SETSCHED
 #define ACE_LACKS_STRRECVFD
 #define ACE_LACKS_WRITEV
+#define ACE_LACKS_MALLOC_H /* netbsd's just includes stdlib.h */
+#define ACE_LACKS_MEMORY_H /* netbsd's just includes string.h */
 #define ACE_LACKS_READV
-#define ACE_LACKS_SYSCONF
-#define ACE_LACKS_GETOPT
 /* below refers to fcntl style locking */
 #define ACE_LACKS_FILELOCKS
 
 #define ACE_LACKS_REALPATH
-#define ACE_HAS_CONST_CHAR_SWAB
 #define ACE_HAS_CONSISTENT_SIGNAL_PROTOTYPES
 
 /***** Not tied to standards AFAIK ****/
 #define ACE_LACKS_MADVISE /* paging optimization not needed with INTEGRITY */
 #define ACE_LACKS_INTTYPES_H
 #define ACE_LACKS_SYS_RESOURCE_H
-#define ACE_LACKS_SYS_WAIT_H
 #define ACE_LACKS_SEARCH_H
 #define ACE_LACKS_SYS_IPC_H
 #define ACE_LACKS_SYS_SEM_H
-#define ACE_LACKS_PWD_H
 #define ACE_LACKS_STROPTS_H
-#define ACE_LACKS_DLFCN_H
-#define ACE_LACKS_REGEX_H
-#define ACE_LACKS_POLL_H
 #define ACE_LACKS_SYS_SHM_H
-#define ACE_LACKS_TERMIOS_H
```

Comment:
Are there maybe any other parts that could be cleaned because some lacks macros are not used anymore (integrity maybe the last to need a certain workaround)?

Reply:
I checked and no further code needs to be cleaned up.